1-getting-started-lessons/1-intro-to-programming-languages/translations/README.ta.md

```diff
@@ -0,0 +1,198 @@
+# வர்த்தகத்தின் நிரலாக்க மொழிகள் மற்றும் கருவிகள் அறிமுகம்
+
+இந்த பாடம் நிரலாக்க மொழிகளின் அடிப்படைகளை உள்ளடக்கியது. இங்கே உள்ளடக்கப்பட்டுள்ள தலைப்புகள் இன்று பெரும்பாலான நவீன நிரலாக்க மொழிகளுக்கு பொருந்தும். 'வர்த்தககருவிகள்' பிரிவில், நீங்கள் ஒரு டெவலப்பராக உங்களுக்கு உதவும் பயனுள்ள மென்பொருள் பற்றி அறிந்து கொள்வீர்கள்.
+
+![அறிமுக நிரலாக்கம்](/sketchnotes/webdev101-programming.png)
+> ஸ்கெட்ச்நோட் [டோமோமி இமுரா](https://twitter.com/girlie_mac)
+
+## விரிவுரைக்கு முந்தைய வினாடி வினா
+[விரிவுரைக்கு முந்தைய வினாடி வினா](https://happy-mud-02d95f10f.azurestaticapps.net/quiz/1)
```

Comment:
Now you can start adding ?loc=ta to all the quizzes since you did that work prior!